src/result.jl

```diff
@@ -55,6 +55,25 @@ Return a vector `group` such that for every color `c`, `group[c]` contains the i
 """
 function row_groups end
 
+"""
+    ncolors(result::AbstractColoringResult)
+
+Return the number of different colors used to color the matrix.
+
+For bidirectional partitions, this number is the sum of the number of row colors and the number of column colors.
+"""
+function ncolors(res::AbstractColoringResult{structure,:column}) where {structure}
+    return maximum(column_colors(res))
```

Comment:
Can we not use `length(res.group)`?
It should be a little bit cheaper.

Reply:
Sure